packages/fxa-auth-db-mysql/db-server/index.js

```diff
@@ -126,6 +126,10 @@ function createServer(db) {
   );
   api.post('/account/:id/locale', withIdAndBody(db.updateLocale));
   api.get('/account/:id/sessions', withIdAndBody(db.sessions));
+  api.put(
```

Comment:
I think this should be a PATCH instead of PUT given we're only updating a single row, no?

Reply:
I have to double check if restify supports this, I think technically either is fine.